packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/delete.html

```diff
@@ -0,0 +1,15 @@
+<oui-modal
+    data-title="{{:: 'dedicatedCloud_datacenter_backup_delete_title' | translate }}"
+    data-on-dismiss="$ctrl.goToBackup()"
+    data-primary-action="$ctrl.delete()"
+    data-secondary-action="$ctrl.goToBackup()"
+    data-type="warning"
+    data-primary-label="{{:: 'dedicatedCloud_datacenter_backup_delete_confirm' | translate }}"
+    data-secondary-label="{{:: 'dedicatedCloud_datacenter_backup_delete_cancel' | translate }}"
+    data-loading="$ctrl.deleting"
```

Comment:
Use `primary-disabled` to check if the backup is active so that the user will not be able to click the button. Though this is handled in the javascript side, it might give the user an impression that nothing happens when the confirm button is clicked :)